tests/integration/remote-fetch/remote-fetch-page.spec.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+/// <reference types="cypress" />
+
+import { CookieUtils, HousekeepingUtils } from "@wso2/identity-cypress-test-base//utils";
+import { RemoteConfigurationPage } from "./page-objects/remote-fetch-page";
+import { v4 as uuidv4 } from "uuid";
+
+const USERNAME: string = Cypress.env("TENANT_USERNAME");
+const PASSWORD: string = Cypress.env("TENANT_PASSWORD");
+const SERVER_URL: string = Cypress.env("SERVER_URL");
+const PORTAL: string = Cypress.env("CONSOLE_BASE_URL");
+const TENANT_DOMAIN: string = Cypress.env("TENANT_DOMAIN");
+
+describe("ITC-3.0.0 - [remote-configuration] - Remote configuration.", () => {
+
+    const remoteFetchConfigurationPage: RemoteConfigurationPage = new RemoteConfigurationPage();
+
+    before(() => {
+        HousekeepingUtils.performCleanUpTasks();
+        cy.login(USERNAME, PASSWORD, SERVER_URL, PORTAL, TENANT_DOMAIN);
+    });
+
+    beforeEach(() => {
+        CookieUtils.preserveAllSessionCookies();
+    });
+
+    context("ITC-3.1.0 - [remote-configuration] - Remote configuration page.", () => {
+
+        it("CDS_3.1.1 - After User login and navigate to remote configuration page", function () {
```

Comment:
Use arrow functions here. And please check the test descriptions. `CDS_3.1.1` -> `ITC-3.1.1`

Reply:
Fixed with commit bb8180e